src/components/ToolTip/index.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import 'test-util/setup';
+import { shallow } from 'enzyme';
+import { wrapTooltip } from './index';
+
+const buttonText = 'children';
+const defaultProps = {
+  text: 'text',
+  children: <button>{buttonText}</button>,
+};
+
+const setup = (props = {}) => {
+  const testProps = { ...defaultProps, ...props };
+  return shallow(wrapTooltip(testProps.children, testProps.text));
+};
+
+describe('ToolTip', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = setup();
+  });
+
+  describe('Render', () => {
+    it('without errors', () => {
+      expect(wrapper.length).toBe(1);
+    });
+  });
```

Comment:
just realised: can you add tests to check that the `text` and the `children` have rendered?